tests/phpunit/tests/functions/wpFuzzyNumberMatch.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * Test wp_fuzzy_number_match().
+ *
+ * @group functions.php
+ * @covers ::wp_fuzzy_number_match
+ */
+class Tests_wpFuzzyNumberMatch extends WP_UnitTestCase {
+
+	/**
+	 * @dataProvider data_wp_fuzzy_number_match
+	 *
+	 * @param $expected
+	 * @param $actual
+	 * @param $precision
+	 * @param $result
```

Comment:
These should have types and descriptions to help testers.

Suggest copying the same entries from [here](https://github.com/WordPress/wordpress-develop/blob/cab053271e272a21b4d140e50ac0d6a78094ecd5/src/wp-includes/functions.php#L8420).